include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -617,6 +617,36 @@ class FEEvaluationBase
     AlignedVector<std::array<T, VectorizedArrayType::size()>> &array,
     const std::array<T, VectorizedArrayType::size()> &         value) const;
 
+  /**
+   * The same as above, just for faces.
+   */
+  VectorizedArrayType
+  read_face_data(const AlignedVector<VectorizedArrayType> &array) const;
```

Comment:
As it is now, these functions could be moved to `FEEvalData`. But I would like to keep them in `FEEvalBase`, since I plan to add `initialize_*_data_vector()` functions in a follow-up PR. These functions will need access to `MatrixFree`, which is not a member of `FEEvalData`.